app/src/main/res/layout/fragment_skeletons.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="@style/App.Story"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="top"
+    android:orientation="vertical"
+    tools:ignore="HardcodedText"
```

Comment:
same as for the compose story, rather than adding the ignore here could you please create string resources? we still have some stories with hardcoded strings, but are trying to move towards resources to showcase good practises

Reply:
ok